tests/acceptance/addon-dummy-generate-test.js

```diff
@@ -272,6 +275,13 @@ describe('Acceptance: ember generate in-addon-dummy', function() {
       assertFile('tests/dummy/app/templates/foo/bar.hbs', {
         contains: '{{outlet}}'
       });
+      assertFile('tests/dummy/app/router.js', {
+        contains: [
+          "this.route('foo', function() {" + EOL,
```

Comment:
I'm wondering if using `EOL` in tests works with code generated by ember-cli/ember-router-generator. I'm not seeing `EOL` included in anything that's created with `recast`.